Dockerfile.production

```diff
@@ -1,16 +1,17 @@
-FROM node:12.18.0-alpine3.11 AS node-modules-builder
+FROM node:12.18.4-alpine3.12 AS node-modules-builder
 # node-modules-builder stage installs/compiles the node_modules folder
+# Python version must be specified starting in alpine3.12
 RUN apk update && apk upgrade && \
     apk --no-cache add --virtual native-deps \
-    g++ gcc libgcc libstdc++ linux-headers autoconf automake make nasm python git curl && \
+    g++ gcc libgcc libstdc++ linux-headers autoconf automake make nasm python2 git curl && \
```

Comment:
What is this python dependency used for anyways? I don't see any need for it in the prod dockerfile, since we don't use pip here

Reply:
node-gyp requires python dependency; there was a note in the troubleshooting guide that node-gyp only supports python 2 but that this is no longer the [case](https://github.com/nodejs/node-gyp); I've also updated the troubleshooting guide